.storybook/main.js

```diff
@@ -4,6 +4,24 @@ module.exports = {
   stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
   addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
   webpackFinal: (config) => {
+    /**
+     * CSS handling, specifically overriding postcss loader
+     */
+    // Find the only Storybook webpack rule that tests for css
+    const cssRule = config.module.rules.find((rule) =>
```

Comment:
this is some sort of postcss conflict because storybook wants 7 and tailwind wants 8